config/locales/en.yml

```diff
@@ -59,6 +59,8 @@ en:
         no_group: No Group
         title_label: Name
         update_btn_submitting: :thredded.form.update_btn_submitting
+        locked:
```

Comment:
Please used the same format as for the other fields.
You can update the locale files by running:

```bash
i18n-tasks mv thredded.messageboard.form.locked.label thredded.messageboard.form.locked_label
```

Reply:
The only other place `Locked` is used is for topics, and that has 2 child nodes `label` and `message` so this was actually following the format. Do you still want me to push the change you suggested?

Reply:
Well in this case the key is under `messageboard.form` but OK either way. We may want to add a message on the form explaining how a locked messageboard behaves later.

Reply:
Ok